modules/op-release-notes-1-13.adoc

```diff
@@ -0,0 +1,141 @@
++
+:FeatureName: using remote tasks on remote pipelines and overriding tasks
+include::snippets/technology-preview.adoc[]
+
+* With this update, when using {pac}, you can view improved information about CI pipeline runs in the Git repository provider. This information includes the namespace and the associated pipeline run.
+
+[id="breaking-changes-1-13_{context}"]
+== Breaking changes
+
+* Before this update, in {pac}, when using a policy group, users who were not a part of policy groups and not allowed explicitly but allowed to run the CI  (via org ownership or otherwise) could sometimes execute pipeline runs by creating events such as pull requests or by entering commands such as `ok_to_test`. With this update, if policy groups are configured, only users that are added to the required policy groups can execute pipeline runs, and users that are a part of the owner organization but not configured in policy groups cannot execute pipeline runs.
+
+[id="known-issues-1-13_{context}"]
+== Known issues
+
+[id="fixed-issues-1-13_{context}"]
+== Fixed issues
+
+* Before this update, some secrets that pipeline runs use for Git authentication could be deleted from the cluster during a cleanup. With this update, a secret is deleted only when all pipeline runs that use it are deleted.
+
+* Before this update, in cases where multiple secrets shared the same prefix and were logged using the git interface, sometimes the concealing process started with a shorter secret, and a part of a longer secret could be displayed in the logs. With this update, when concealing secrets in logs, the process now starts from the longest secret, ensuring that no part of any secret is displayed in the logs.
+
+* Before this update, if you specified a `results` spec for a pipeline, the pipeline run could wrongly fail with a mismatched types error. With this update, if you specify a `results` spec for a pipeline, the results provided by the pipeline are correctly processed.
+
+* Before this update, when {tekton-chains} was configured with KMS as Hashicorp Vault, the pod started crashing if there was an underlying error while connecting to Vault. This has now been fixed and the error is now recorded in the {tekton-chains} controller log.
+
+* Before this update, when using {tekton-chains}, if you configured the `storage.oci.repository` parameter, errors were reported in the Tekton Chains controller log. With this update, the `storage.oci.repository` parameter is processed correctly.
+
+* Before this update, when {tekton-chains} was configured with the Hashicorp Vault KMS and there was an issue with the connection to Vault, the {tekton-chains} controller pod could crash. With this update, the errors are processed and logged on the {tekton-chains} controller log.
```

Comment:
@PuneetPunamiya Is this and line number 137 same?

Reply:
@VeereshAradhya I don't think they are the same. Apart from being Chains, there's little in common, storage is storage and KMS is KMS.